tasklauncher-cloudfoundry-app-dependencies/pom.xml

```diff
@@ -13,15 +13,15 @@
     <parent>
         <artifactId>spring-cloud-dependencies-parent</artifactId>
         <groupId>org.springframework.cloud</groupId>
-        <version>1.2.1.RELEASE</version>
+        <version>1.2.1.BUILD-SNAPSHOT</version>
```

Comment:
Good point. I think we can apply this change across all the app starters may be?